hal/armv7a/imx6ull/_init.S

```diff
@@ -43,6 +43,7 @@ _vector_table:
 
 
 .org 0x400, 0x0
+#ifdef PLO_NOR_BOOT
```

Comment:
IVT needs to be @0x400 for NAND boot? It's a shame we cannot have universal plo for both NAND and NOR boot